app/components/App.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import {Nav, NavItem} from 'react-bootstrap'
 
-import GlobalScoreboard from 'components/GlobalScoreboard'
+import GlobalScoreboard     from 'components/GlobalScoreboard'
```

Comment:
I would suggest not doing this kind of alignment because it's not automatic and not all of use do it, we'll end up with a mix of alignments and non-alignments, I've seen it many times, it's not maintainable.

Reply:
I think we need to talk about this in the next meeting.

Reply:
Post the question in slack.